src/types.h

```diff
@@ -209,7 +218,8 @@ class SizedType
   bool IsPrintableTy()
   {
     return type != Type::none && type != Type::stack_mode &&
-           (!IsCtxAccess() || is_funcarg); // args builtin is printable
+           (!IsCtxAccess() || is_funcarg) &&
+           type != Type::timestamp_mode; // args builtin is printable
```

Comment:
More readable and also the comment is well-placed:
```suggestion
           type != Type::timestamp_mode &&
           (!IsCtxAccess() || is_funcarg); // args builtin is printable
           
```